src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -224,4 +225,13 @@ trait MarathonConf
     descr = "The timeout for preparing the Marathon instance when elected as leader.",
     default = Some(3 * 60 * 1000L) //3 minutes
   )
+
+  lazy val maxQueuedRootGroupUpdates = opt[Int]("max_queued_root_group_updates",
```

Comment:
Ack for renaming the vars.

we haven't renamed the new options for the TaskTrackerActor "internal*" on the command line. They are marked as "hidden" though and have "INTERNAL TUNING PARAMETER" as description.

The command line parameters get quite long... Anyway, either way we go we should treat all new internal options.